Maestro/subscriptions.json

```diff
@@ -41,6 +41,22 @@
     {
       "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt",
       "handlers": [
+        // This handler will bring the Latest CoreFX release/1.0.0 build into CoreFX release/1.0.0
```

Comment:
I believe it’s used to test against the latest prerelease packages and to let projects choose to use the prerelease version to get access to API changes. This way they keep up to date with the latest prerelease. Whether or not that’s needed in this case (re: email), I don’t know. /cc @weshaggard 

Were you thinking this should be clarified in the json comment? I'm trying to keep the comments the same format so that they're easy to visually scan, and I'm hoping that the pointer to the script that it runs can explain it further.